### ISSUE TITLE
Can't find devices 

### ISSUE THREAD

**Issue by looooool11111** [innoveit/react-native-ble-manager#1273]
**Describe the bug**
Bluetooth devices cannot be searched on Android 11 devices, but can be used normally on devices larger than Android 11

**Screenshots**
Screenshots
AndroidManifest.xml
![image](https://github.com/user-attachments/assets/254c4571-f656-4ca4-a1da-b4d449f71d0f)

startScan
![image](https://github.com/user-attachments/assets/7b0b8bad-5e97-4f0d-b576-c64ac1a1ac69)

Can't find devices
![image](https://github.com/user-attachments/assets/91f2e597-85fc-4686-af3f-00d5a518877d)

init
![image](https://github.com/user-attachments/assets/3a4459eb-ba01-4ca7-bc41-c6b542052d72)

**Smartphone (please complete the following information):**
 - OS: [e.g. iOS8.1]:android 11
 - react-native-ble-manager:11.5.5
 - react-native version:0.73.4

**Comment by lucaswitch** [comment]
Thats actually a problem with runtime permissions on Android. I can confirm that is working as expected on Android 11.

**Comment by looooool11111** [comment]
> Thats actually a problem with runtime permissions on Android. I can confirm that is working as expected on Android 11.

yeah,I checked the device's permission settings and found that my application has obtained Bluetooth related permissions, but has not requested to use them

**Comment by lucaswitch** [comment]
> > Thats actually a problem with runtime permissions on Android. I can confirm that is working as expected on Android 11.
> 
> yeah,I checked the device's permission settings and found that my application has obtained Bluetooth related permissions, but has not requested to use them

Did you was able to solve it?

**Comment by looooool11111** [comment]
> Did you was able to solve it?

not yet
